.vale/styles/RedHat/TermsSuggestions.yml

```diff
@@ -4,38 +4,18 @@ ignorecase: false
 level: suggestion
 link: https://redhat-documentation.github.io/vale-at-red-hat/docs/main/reference-guide/termssuggestions/
 message: "Depending on the context, consider using '%s' rather than '%s'."
-# source: "https://redhat-documentation.github.io/supplementary-style-guide/#glossary-terms-conventions; IBM - Appendix C. Word usage, p. 300"
 action:
   name: replace
 swap:
-  "(?<!,) which": ", which (non restrictive clause preceded by a comma)|that (restrictive clause without a comma)"
-  "(?<!by) using": by using|that uses
+  "(?<!,) which": ", which| that"
```

Comment:
Sometimes folks incorrectly precede _that_ with a comma, for example, `, that` Do we also need an items here to cover that scenario? 
Ignore this comment if it's covered elsewhere. Thanks.

Reply:
oops. Yes. Added this.